test/dxf/parser_rspec.rb

```diff
@@ -0,0 +1,16 @@
+require 'dxf/parser'
+
+RSpec.describe DXF::Parser do
+  it 'must read file containing Polyline' do
+    parser = File.open('test/fixtures/AD_include_polyline.dxf', 'r') {|f| DXF::Parser.new.parse(f) }
+    expect(parser.entities.length).to eq(6)
+    first = parser.entities.first
+    expect(first).to be_a DXF::Line
+  end
+  it 'must read file NOT containing Polyline' do
+    parser = File.open('test/fixtures/AD_only_lines.dxf', 'r') {|f| DXF::Parser.new.parse(f) }
+    expect(parser.entities.length).to eq(5)
+    first = parser.entities.first
+    expect(first).to be_a DXF::Line
+  end
+end
```

Comment:
this test was generated as I could not run the MiniTest that were already on the library on my local environment, it cover a file with Polyline as the new provided by AD, and a file with lines as are the current files being used.

On local was tested with the full DXF files, for this testing I cropped the files to only 5/6 entities.